app/src/main/java/com/teamagam/gimelgimel/app/Alerts/view/AlertsSubcomponent.java

```diff
@@ -22,110 +19,76 @@
-
-        private ValueAnimator mColorAnimation;
-        private final int mToolbarOriginalColor;
-        private final int mToolbarAnimationPeakColor;
-
-        private boolean mIsAnimating;
-
-
-        private PeakColorToolbarAnimator(Toolbar toolbar, Context context) {
-            mToolbarOriginalColor = ContextCompat.getColor(context, R.color.colorPrimaryDark);
-            mToolbarAnimationPeakColor =
-                    ContextCompat.getColor(context, R.color.alerts_informing_peak_color);
+    private class AlertDisplayer implements AlertsViewModel.AlertDisplayer {
 
-            initializeAnimation(toolbar, mToolbarAnimationPeakColor, mToolbarOriginalColor);
+        private boolean mIsShowingAlert;
+        private Alert mCurrentAlert;
 
-            mIsAnimating = false;
+        public AlertDisplayer() {
+            mIsShowingAlert = false;
         }
 
-        @Override
-        public void animateActionBar() {
-            mIsAnimating = true;
-            mColorAnimation.start();
-        }
-
-        @Override
-        public void stopActionBarAnimation() {
-            mColorAnimation.cancel();
-            mToolbar.setBackgroundColor(mToolbarOriginalColor);
-            mIsAnimating = false;
-        }
+        public void showAlert(String title, String description) {
+            mCurrentAlert = getBaseAlerter()
+                    .setTitle(title)
+                    .setText(description)
+                    .show();
 
-        @Override
-        public boolean isAnimating() {
-            return mIsAnimating;
+            mIsShowingAlert = true;
         }
 
-        private void initializeAnimation(final Toolbar toolbar, int from, int to) {
-            mColorAnimation = ObjectAnimator.ofObject(toolbar, "backgroundColor",
-                    new ArgbEvaluator(), from, to);
-
-            mColorAnimation.setDuration(ANIMATION_DURATION_MS);
-            mColorAnimation.setStartDelay(ANIMATION_START_DELAY_MS);
-            mColorAnimation.setRepeatCount(ValueAnimator.INFINITE);
-            mColorAnimation.setRepeatMode(ValueAnimator.REVERSE);
+        public void hideAlert() {
+            if (mCurrentAlert != null) {
+                mCurrentAlert.hide();
+                mIsShowingAlert = false;
+            }
         }
-    }
 
-    private class ToolbarTitleSetter implements AlertsViewModel.ToolbarTitleSetter {
-        @Override
-        public void setTitle(String title) {
-            mToolbar.setTitle(title);
+        public boolean isShowingAlert() {
+            return mIsShowingAlert;
         }
 
-        @Override
-        public void restoreDefault() {
-            mToolbar.setTitle(mToolbarOriginalTitle);
+        private Alerter getBaseAlerter() {
+            return Alerter.create(mActivity)
+                    .enableIconPulse(true)
+                    .enableInfiniteDuration(true)
+                    .setBackgroundColor(R.color.alerts_informing_peak_color)
```

Comment:
Change the resource name (and its comment, since it's not used for animation anymore)
alerter_background_color?
alert_notifying_background_color?